src/dmd/cli.d

```diff
@@ -566,6 +566,9 @@ dmd -cov -unittest myprog.d
         Option("vgc",
             "list all gc allocations including hidden ones"
         ),
+        Option("vnrvo",
+            "list all functions that return using named return value optimization"
+        ),
```

Comment:
If we expose it publicly, we should probably add it a full changelog entry for it?

Reply:
Probably, I also put in a link that doesn't exist yet either.